searches/quick_select.py

```diff
@@ -26,6 +26,16 @@ def _partition(data, pivot):
 
 
 def quickSelect(list, k):
```

Comment:
Please change the function signature to be `def quick_select(items: list, index: int):`  This does the following...
1. Use a Pythonic function name instead of a function name from JavaScript.
2. Change the parameter __list__ to be __items__ because __list__ is a Python builtin data type so it is not a suitable variable name.
3. Add typehints for the list and the int
4. Change the parameter __k__ to be __index__ because that name is more self-documenting.